Logger/Employee.cs

```diff
@@ -0,0 +1,44 @@
+﻿
+namespace Logger;
+
+public record Employee : IEntity
+{
+    private FullName _name;
+    public int Eid { get; set; }
+
+    //TODO: explain why this is implemeted implicitly
+    public Guid Id { get; init; }
+
+    // implemented implicitly since Name is a frequently-accessed
+    public string Name
+    {
+        get => _name.ToString();
+        set => _name = new(value);
+    }
```

Comment:
```suggestion
    public string Name { get => _name.FirstName + " " + _name.LastName; }
```
Use a calculated property instead. This would be even easier with a `Person` object as some reviewers have mentioned